apollo-ios/Sources/Apollo/IncrementalGraphQLResponse.swift

```diff
@@ -27,20 +27,73 @@ final class IncrementalGraphQLResponse<Operation: GraphQLOperation> {
   private let base: AnyGraphQLResponse
 
   public init(operation: Operation, body: JSONObject) throws {
-    guard let path = body["path"] as? [JSONValue] else {
-      throw ResponseError.missingPath
-    }
+    guard let path = body["path"] as? [JSONValue] else { throw ResponseError.missingPath }
```

Comment:
Why did you remove a bunch of new line indentation throughout this PR?

Reply:
Just personal preference, I'll revert and change my new code to match the style it was before.

My preferences:
* If a function call can fit on a single line within the 120 column width then make it one line.
* If there is only a single instruction between `{` and `}` put it all on a single line too provided it fits within the 120 column limit.